requirements.freeze.txt

```diff
@@ -0,0 +1,237 @@
+#blas: replaced with apt/brew install depending on OS
+#openssl: not found but "ssl" is part of the standard library set
+#python.app
+alabaster==1.0.0
+appnope==0.1.4
+appscript; platform_system == "Darwin"
+asn1crypto==1.5.1
+astroid==3.2.4
+astropy==6.1.2
+atomicwrites==1.4.1
+attrs==23.2.0
+babel==2.15.0
+backcall==0.2.0
+backports.os==0.1.1
+backports.shutil_get_terminal_size==1.0.0
+backports.tarfile==1.2.0
+beautifulsoup4==4.12.3
+bitarray==2.9.2
+bkcharts==0.2
+bleach==6.1.0
+blosc==1.11.2
+bokeh==3.5.1
+boto==2.49.0
+bottleneck==1.4.0
+certifi==2024.7.4
+cffi==1.16.0
+chardet==5.2.0
+click==8.1.7
+cloudpickle==3.0.0
+clyent==1.2.1
+colorama==0.4.6
+contextlib2==21.6.0
+cryptography==43.0.0
+cycler==0.12.1
+cython==3.0.10
+cytoolz==0.12.3
+dash-core-components==2.0.0
+dash-daq==0.5.0
+dash-html-components==2.0.0
+dash-renderer==1.9.1
+dash-table==5.0.0
```

Comment:
I added requirements.freeze.txt which is not a full freeze, just lists the explicit packages in the regular requirements.txt, but with versions explicitly specified.  In case we have an issue later, or need a deterministic build.